connaisseur/notary_api.py

```diff
@@ -23,6 +23,12 @@ def health_check(host: str):
     if not host:
         return False
 
+    # The ACR does not expose the notary health endpoint: https://github.com/Azure/acr/issues/475
```

Comment:
would really write out ACR as Azure Container Registry here and start with that:
`Set to true if you are using Azure Container Registry (ACR). (...)`

Otherwise, people may be confused by this option.

Reply:
You probably mean this: https://github.com/sse-secure-systems/connaisseur/blob/db06c5e834bd75a78e7567c5b76b8a0e7fcf728b/helm/values.yaml#L48
Whoever looks into the actual implementation instead of the config file can know what ACR is at latest from following the link